packages/chat-stateful-client/src/ResourceDownloadQueue.ts

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+/* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
+import { ChatContext } from './ChatContext';
+/* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
+import { ChatError } from './ChatClientState';
+/* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
+import { ChatMessageWithStatus } from './types/ChatMessageWithStatus';
+/* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
+import { CommunicationTokenCredential } from '@azure/communication-common';
```

Comment:
Could you update `import` to `import type` as we probably can import only type instead of the whole object here?

It helps bundlers to understand a little bit better what resources need to be included

Reply:
Done